ufc/tests/test-case-flag-with-empty-string.json

```diff
@@ -0,0 +1,143 @@
+{
+  "flag": "string-variations",
+  "variationType": "STRING",
+  "defaultValue": {},
+  "subjects": [
+    {
+      "subjectKey": "alice",
+      "subjectAttributes": {},
+      "assignment": {
```

Comment:
this is the expected assignment, which I imagine would be the empty string. Look at `test-case-new-user-onboarding-flag.json` as an example test case for a string-valued flag